src/main/java/sirius/biz/protocol/DelegateJournalData.java

```diff
@@ -43,6 +43,9 @@ public class DelegateJournalData extends Composite {
     @Transient
     private volatile boolean silent;
 
+    @Transient
+    private boolean logCreation;
```

Comment:
Does this also have to be volatile like the silent flag above?

Reply:
🤷‍♂️


i'll check